### PR TITLE
RELATED: RAIL-2654, RAIL-2658 add missing examples

### PR DESCRIPTION
**Pull request by no23reason**
There were some examples present in SDK7 examples missing here

- MeasureValueFilter with treat as zero
- Custom Visualization with recharts

Also existing examples were improved

- Missing show code button was added to GeoPushpinDrilling example
- Dynamic sorting examples' wording and UX was improved

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by no23reason** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/599/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2193/ : SUCCESS in 8m 55s

File: examples/sdk-examples/src/examples/execution/ExecuteWithCustomVisualizationExample.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                            dataKey={value.id}
+                            name={value.measureTitle()}
+                            fill={colors[index]}
+                        />
+                    );
+                });
+
+                const data = slices?.map((slice) => ({
+                    label: slice.sliceTitles()[0],
+                    ...slice.dataPoints().map((p) => p.rawValue),
```
Comment (lupko) [inline examples/sdk-examples/src/examples/execution/ExecuteWithCustomVisualizationExample.tsx:51]: is it possible to use formatted data here? or the chart only supports numbers?

Comment (no23reason) [reply examples/sdk-examples/src/examples/execution/ExecuteWithCustomVisualizationExample.tsx:51]: I'm not sure, let me check

Comment (no23reason) [reply examples/sdk-examples/src/examples/execution/ExecuteWithCustomVisualizationExample.tsx:51]: Nope, the library only supports numbers.

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/303/ : FAILURE in 3m 04s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/559/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2201/ : SUCCESS in 9m 53s

**Pull request merged**